### PR TITLE
feat: Add date attribute to flag

### PR DESCRIPTION
**Pull request by paultranvan**
This will be useful to set up a custom date for the files query, so the
indicator can be more easily tested.

### CODE REVIEW

File: src/drive/targets/services/dacc.js [hunk 1]
```diff
@@ -35,11 +35,17 @@ export const run = async () => {
...
+    maxFileDateQuery
   } = daccFileSizeFlag
-  const aggregationDate = endOfMonth(subMonths(new Date(), 1))
+
+  const aggregationDate =
+    maxFileDateQuery || endOfMonth(subMonths(new Date(), 1))
 
   const sizesBySlug = await aggregateFilesSize(client, aggregationDate)
+  if (Object.keys(sizesBySlug).length < 1) {
+    log('info', `No files found to aggregate with date ${aggregationDate}`)
```
Comment (trollepierre) [inline src/drive/targets/services/dacc.js:47]: I would expect 2 tests for this kind of PRs, but I wonder if there are drawbacks testing service. I asked a question to @Crash-- , and according to his answer, I will approve or suggest the 2 tests

Comment (trollepierre) [reply src/drive/targets/services/dacc.js:47]: => https://github.com/cozy/cozy-drive/commit/ced3151db01869145235133e2a1bd74b972c0918

you can cherry pick this commit (and modify to comply with the business need) :-)

Comment (paultranvan) [reply src/drive/targets/services/dacc.js]: That's very nice, I slightly adapted it with [25d28d6](https://github.com/cozy/cozy-drive/pull/2499/commits/25d28d6b355233d0315a1029ca674df006785f8d) :)

**Review by trollepierre — APPROVED** [review]
test is failing 🙏

**Comment by paultranvan** [comment]
Acutally, the tests were failing both because of master and a small change in the PR. It's fixed :)

**Pull request merged**